API/companyapi/api/admin.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from api.models import *
```

Comment:
Separate these imports with a single line as one is related to your project and other is a django import.